pom.xml

```diff
@@ -36,7 +36,7 @@
     <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <datastax-java-driver.version>4.5.0</datastax-java-driver.version>
+    <datastax-java-driver.version>4.6.1</datastax-java-driver.version>
```

Comment:
The newest driver didn't introduce any problems in the native build?

Reply:
No, with the substitutions in this PR there wasn't a problem at all.  It's as if the native code just always fails to load, so nothing winds up using it (since all calls to it in the current driver code are wrapped by Native.is*Available() checks).

JAVA-2663 will provide the real fix here (via Graal-friendly Native methods)... this just allows us to change driver versions freely without worrying about breaking things.